editor/gui/scene_tree_editor.cpp

```diff
@@ -344,7 +344,9 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 		}
 
 		if (p_node->is_unique_name_in_owner()) {
-			item->add_button(0, get_editor_theme_icon(SNAME("SceneUniqueName")), BUTTON_UNIQUE, p_node->get_owner() != EditorNode::get_singleton()->get_edited_scene(), vformat(TTR("This node can be accessed from within anywhere in the scene by preceding it with the '%s' prefix in a node path.\nClick to disable this."), UNIQUE_NODE_PREFIX));
+			bool is_enabled = p_node->get_owner() == EditorNode::get_singleton()->get_edited_scene();
+			String button_text = vformat(TTR("This node can be accessed from anywhere within the scene it belongs to by using the '%s' prefix in the node path%s."), UNIQUE_NODE_PREFIX, is_enabled ? "\nClick to disable this." : "");
```

Comment:
```suggestion
			String button_text = vformat(TTR("This node can be accessed from anywhere within the scene it belongs to by using the '%s' prefix in the node path.%s"), UNIQUE_NODE_PREFIX, is_enabled ? ("\n" + TTR("Click to disable this.")) : "");
```
The additional text needs to be translated, and the period should still be at the end of the first sentence